src/Adapter/MSTest.CoreAdapter/Execution/TestMethodRunner.cs

```diff
@@ -84,6 +128,26 @@ internal UnitTestResult[] Execute()
             string inittestContextMessages = string.Empty;
 
             UnitTestResult[] result = null;
+
```

Comment:
a related question: How can someone leverage extensibility to achieve this test level skipping?